.github/workflows/drata_iac_scan.yml

```diff
@@ -10,10 +10,10 @@ jobs:
       - uses: actions/checkout@v4
 
       - name: Drata Action
-        uses: drata/compliance-as-code-action@v0.0.98
+        uses: drata/compliance-as-code-action@v0.0.99
```

Comment:
We're up to 102 at the moment https://github.com/drata/compliance-as-code-action/releases/tag/v0.0.102

Reply:
@bnicoll-drata is that not getting automatically updated here https://github.com/marketplace/actions/drata-compliance-as-code-action?version=v0.0.99? That's where I looked

Reply:
Good point, I don't know how that page gets updated, but the repo release page is the most up-to-date